src/BinaryParsers/PEBinary/ProgramDatabase/ObjectModuleDetails.cs

```diff
@@ -224,28 +224,35 @@ public string GetDialect(out string versionNumber)
         }
     }
 
+    /// <summary>
+    /// Information about the CV_CFL_LANG enumeration, 
```

Comment:
>Information [](http://example.com/codeflow?start=8&length=11)

This class is an implementation of the enumeration not information about the enumeration (though you do have some code comments that won't be visible to anyone viewing your summary comment via intellisense.